install_build_dependencies.sh

```diff
@@ -244,6 +244,57 @@ elif [ -f /etc/os-release ] && grep -q "void" /etc/os-release; then
         enchant2-devel \
         `# samples` \
         json-c++
+elif [ -f /etc/os-release ] && grep -q "alpine" /etc/os-release; then
+    #Alpine Linux
+    apk --no-cache add \
+        `# for python3-pip` \
+		ca-certificates \
+		file \
+		`# build tools` \
+		build-base \
+		ninja-is-really-ninja \
+		scons \
+		ccache \
+		cmake \
+		`# to find dependencies` \
+		pkgconf \
+		`# to determine product version via git` \
+		git \
+		`# to check bash scripts for correctness` \
+		shellcheck \
+		`# to build and check pip packages` \
+		patchelf \
+		fdupes \
+		`# main openvino dependencies` \
+		onetbb-dev \
+		py3-tbb \
+		pugixml-dev \
+		`# OpenCL for GPU` \
+		opencl-icd-loader-dev \
+		opencl-dev `#(includes opencl-headers)`\
+		rapidjson-dev \
+		`# GPU plugin dependency` \
+		libva-dev
+		`# For TF FE saved models` \
+		snappy-dev
+		`# For Python API` \
+		py3-pip `#(includes py3-setuptools)`\
+		py3-wheel
```

Comment:
please, add back slashes

Reply:
slashes are still not fixed

Reply:
Yes, I didn't add the changes yet. I'm figuring out which OpenCL ICL to use first.